modules/tiles/src/tileset/helpers/tiles-3d-lod.ts

```diff
@@ -122,6 +122,10 @@ export function getDynamicScreenSpaceError(tileset, distanceToCamera) {
   return 0;
 }
 
+function defined(x: unknown): boolean {
```

Comment:
looks like the `3d-tiles` already defines a `defined` function which behaves differently (only checks undefined, not null). To avoid confusion, how about we define an `isNil` function following lodash's example: `const isNil = (value: any) => value === undefined || value === null;`

Then `!defined(tile.parent)` becomes `isNil(tile.parent)`. To make the ternary expression where we calculate `tileViewDistanceScale` super clear, we might even introduce a lil helper function:

```typescript
const isTilesetRoot = (tile: Tile) => isNil(tile.parent);
```

and then we'll have `const tileViewDistanceScale = isTilesetRoot(tile) ? ...`

Reply:
@justinmanley it looks like the `defined` function has been defined in a few places. In `tileset/helpers/bounding-volume.ts` there is another one that's the same as the one I've written. I copied it form and didn't export it because it's so small and to avoid an import from an unrelated file. 

I'm happy to update if you still think it's necessary?